utils/k8s/k8s.go

```diff
@@ -660,3 +660,22 @@ func (k8s K8S) BlockUntilPodIsUpOrTimeout(pod *core_v1.Pod, timeout time.Duratio
 
 	return k8s.BlockUntilPodIsUpWithContext(ctx, pod)
 }
+
+// CreatePersistentVolumeClaim is use to create the Pesistent Volume Claim for the given name space
```

Comment:
`// CreatePersistentVolumeClaim is use to create the PVC in the given name space`